src/config.js

```diff
@@ -149,6 +151,8 @@ const config: AppConfig = {
   resyncTransactionsTimeout: 5 * ONE_MINUTE,
 
   syncBalancesTimeout: 10 * ONE_SECOND,
+
+  miningDelay: ONE_DAY,
```

Comment:
It looks more like "timeout" (if it is not mined in specified time, it is considered an error), but right now I'm fine with "delay" too